app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    args = command.split()
+    if args[0] != "cp":
+        raise ValueError("Command must start with 'cp'.")
+    command_name, source_file, destination_file, *_ = args
+    if source_file == destination_file:
+        raise ValueError("Source and destination files are the same.")
```

Comment:
The better ordering for operations would be the following
```suggestion
    args = command.split()
    command_name, source_file, destination_file, *_ = args
    
    if command_name != "cp":
        raise ValueError("Command must start with 'cp'.")
    if source_file == destination_file:
        raise ValueError("Source and destination files are the same.")
```
Use an approach that allows you to avoid using indexing and replace it with descriptive variable names where possible.